health_monitor/src/health_monitor.cpp

```diff
@@ -82,19 +82,19 @@ namespace health_monitor
                 new_msg.description = header;
                 new_msg.type = cav_msgs::SystemAlert::SHUTDOWN;
 
-                pnh_->publishSystemAlert(new_msg);
+                ros::CARMANodeHandle::publishSystemAlert(new_msg);
```

Comment:
I received an error when I tried to build the health monitor package:

error: cannot call member function 'void ros::CARMANodeHandle::publishSystemAlert(const SystemAlert&)' without object
ros::CARMANodeHandle::publishSystemAlert(new_msg);
^